src/main/java/com/microsoft/sqlserver/jdbc/SQLServerResource.java

```diff
@@ -643,5 +645,5 @@ static String getResource(String key) {
             {"R_pvkParseError", "Could not read Private Key from PVK, check the password provided."},
             {"R_pvkHeaderError", "Cannot parse the PVK, PVK file does not contain the correct header."},
             {"R_clientCertError", "Reading client certificate failed. Please verify the location of the certificate."},
-            {"R_unassignableError", "The class specified by the {0} property must be assignable to {1}."}};
+            {"R_unassignableError", "The class specified by the {0} property must be assignable to {1}."},};
```

Comment:
Remove this comma at the end

Reply:
I prefer to actually leave the commas for these lists in so when we add lines it's obvious where to add them, and it's more convenient to just copy+past the last line.